src/signature/RsaSha.php

```diff
@@ -52,8 +52,10 @@ class RsaSha extends BaseMethod
     /**
      * {@inheritdoc}
      */
-    public function init()
+    public function __construct($algorithm = null)
     {
+        if($algorithm != null) $this->algorithm = $algorithm;
```

Comment:
It makes sense because subclass defines default value for that parameter. If I create an object from subclass with null value, it is intended to use default value

Reply:
Fixed